### ISSUE TITLE
“fused_multi_tensor is not installed corrected” && ”Cannot use GPU“

### ISSUE THREAD

**Issue by YingjieQu** [xichengeva/DeepGlycanSite#9]
Hello, I make sure the environment has the correct pytorch, pytorch-geometric, cuda versions:
Python 3.9.16

Name                    Version                   Build  Channel
pytorch-lightning         1.9.3                    pypi_0    pypi
torch                     2.0.0+cu118              pypi_0    pypi
torch-cluster             1.6.3+pt20cu118          pypi_0    pypi
torch-geometric           2.5.3                    pypi_0    pypi
torch-scatter             2.1.1+pt20cu118          pypi_0    pypi
torch-sparse              0.6.18+pt20cu118          pypi_0    pypi
torch-spline-conv         1.2.2+pt20cu118          pypi_0    pypi
torchmetrics              1.4.0.post0              pypi_0    pypi
#======================================================================#

But I am running the `Run inference with only receptor` instance with the following warning:
fused_multi_tensor is not installed corrected
fused_rounding is not installed corrected
fused_layer_norm is not installed corrected
fused_softmax is not installed corrected
Loading pretrained weights from D:\Desktop\code\DeepGlycanSite-main\src\unimol_tools\weights\mol_pre_all_h_220816.pt
GPU available: True (cuda), used: False
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
HPU available: False, using: 0 HPUs
Testing: 0it [00:00, ?it/s]
......
Testing DataLoader 0: 100%|██████████| 1/1 [00:01<00:00,  1.92s/it]
────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
       Test metric             DataLoader 0
────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
        test_loss                   0.0
────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
#======================================================================#

I changed 593 lines of code from `torch.device('cpu')` to `torch.device('cuda'),` and the problem still exists, even though `torch.cuda.is_available()` returns True

I would like to know if there is any impact on the normal use of DeepGlycanSite, and how to use GPU acceleration, Thank you!

**Comment by Xinheng-He** [comment]
Maybe it's a problem with Windows and switching to Linux system can solve it. 
Of note, using GPU is not too much accelerating because it's just an inference process.

**Comment by YingjieQu** [comment]
Thank you for the suggestion. The Windows system is not suitable, so I switched to the Linux system and the warning of running the `Run inference with only receptor` was still there:
fused_multi_tensor is not installed corrected
fused_rounding is not installed corrected
fused_layer_norm is not installed corrected
fused_softmax is not installed corrected
Loading pretrained weights from /raid/quyj/DeepGlycanSite-main/src/unimol_tools/weights/mol_pre_all_h_220816.pt
GPU available: True (cuda), used: False
......
#================================================================================#

but the `Retraining DeepGlycanSite` could use the GPU without the warning above using Linux system.
